src/components/Footer.tsx

```diff
@@ -87,22 +83,16 @@ const Footer = () => {
                 |{" "}
                 <a target="_blank" href={config.testnetUrl}>
                     {t("testnet")}
-                </a>{" "}
-                | <a href="/terms">{t("terms")}</a>
+                </a>
             </p>
             <p>{t("footer")}</p>
-            <p class="version">
-                {t("version")}:{" "}
+            <p class="legal-nav">
+                <a href="/terms">{t("terms")}</a>
+                <a href="/privacy">{t("privacy")}</a>
                 <a
                     target="_blank"
                     href={`${config.repoUrl}/releases/tag/v${__APP_VERSION__}`}>
-                    {__APP_VERSION__}
-                </a>
-                , {t("commithash")}:{" "}
-                <a
-                    target="_blank"
-                    href={`${config.repoUrl}/commit/${__GIT_COMMIT__}`}>
-                    {__GIT_COMMIT__}
```

Comment:
That commit hash and it explicity saying the version is useful for debugging and stays

Reply:
I was the one originally vouching for having the hash there, just that I never used it for debugging. Not once. You? Also we almost never have the situation where we are distributing a version on  boltz.exchange that is not a tagged release. Hence version = commit.

Back in the day things were differently but now I think linking to a version is more than enough.

Reply:
Anyhow I'll play around with it a bit later today trying to make it work without destroying the optics

Reply:
> You?

Yes

 > Also we almost never have the situation where we are distributing a version on boltz.exchange that is not a tagged release. Hence version = commit

We are literally not right now

Reply:
*shouldn't

Reply:
We do it all the time. We need that version field